cluster/node-pools/master-default/userdata.yaml

```diff
@@ -201,6 +201,7 @@ write_files:
 {{- if eq .Cluster.ConfigItems.teapot_admission_controller_inject_environment_variables "true" }}
             - --pod-enable-env-inject
             - --pod-inject-env=_PLATFORM_ACCOUNT={{ .Cluster.Alias }}
+            - --pod-inject-env=_PLATFORM_CLUSTER_ID={{ .Cluster.id }}
```

Comment:
Should be `ID` ([source](https://github.com/zalando-incubator/cluster-lifecycle-manager/blob/master/api/cluster.go#L21)).